nbs/examples/Automatic_Hyperparameter_Tuning.ipynb

```diff
@@ -22,9 +22,11 @@
     " 3. Sample configurations with a search algorithm, train models, and evaluate them on the validation set.\n",
```

Comment:
<p>The default sampler in optuna is the TPESampler (it's the one chosen in the <a href="https://optuna.readthedocs.io/en/stable/reference/generated/optuna.study.create_study.html#optuna.study.create_study" target="_blank">create_study function</a> if the sampler argument is None)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Fixed</p>